tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,465 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Consistent scores\n",
+    "\n",
+    "A \"consistent scoring function\" is a scoring function where a forecaster's expected score will be optimised by following a forecast directive. An example of a forecast directive is to \"predict the mean (i.e., expected) rainfall amount\". Using a consistent scoring function ensures that a forecaster or forecast system developer isn't faced with the dilemma of whether to \n",
+    "\n",
+    "1. produce forecasts (or develop a forecast system) that optimises the expected verification score, or\n",
+    "2. produce an honest forecast (or develop an honest forecast system) that is consistent with the forecast directive. \n",
```

Comment:
It's not that either is necessarily dishonest as such. And, it could be that both the forecast directive and the expected verification score are both sub-optimal in some fashion. Or that the evaluation score is fair and the directive is inappropriate. I think maybe for the second point "2. produce a forecast (or develop a forecast system) that optimises for the forecast directive".